### ISSUE TITLE
Improve Search in UI (Registry and Workflows)

### ISSUE THREAD

**Issue by skrilab** [TracecatHQ/tracecat#429]
**Describe the bug**
The Search could be inproved in UI (Registry and Workflows). Now it seems that search is done only in Action title, but Namespace (integration itslef) field could also be used.

**To Reproduce**
1. Go to 'Workflows'
2. Add new 'Node'
3. Search for 'LDAP'
4. Only one action is returned

**Expected behavior**
Search results should return all actions that are LDAP (integration) related

**Screenshots**
![Xnip2024-10-15_09-22-26](https://github.com/user-attachments/assets/7666051b-a7a6-4ca1-86f1-5e4b433707e6)

![Xnip2024-10-15_09-31-52](https://github.com/user-attachments/assets/0097a7ba-ac28-49ba-b930-e94a5c16f4e5)

**Comment by daryllimyt** [comment]
Agreed. Happy to accept a PR or we can scope it in for sometime this week!
